accounts-db/src/accounts_db.rs

```diff
@@ -1349,16 +1351,55 @@ impl StoreAccountsTiming {
 struct CleaningInfo {
     slot_list: SlotList<AccountInfo>,
     ref_count: u64,
-    /// True for pubkeys mapping to older versions of accounts that should be purged.
-    should_purge: bool,
+}
+
+#[derive(Debug)]
```

Comment:
Will it be better if we combine HashMap and BitVec in one RwLock?
It seems that we always grab the write/read lock of these two at the same time.

Reply:
Yes, it probably would be more efficient to combine them as a tuple in one RwLock, but I think it would be more awkward to use. In the longer term (in subsequent PRs) I would prefer to hide the fields of CleaningMap behind a more elaborate interface so that the implementation details are not exposed to the struct users, and then it would be more convenient to combine these two fields under the same RwLock.